lyluatex.md

```diff
@@ -319,6 +319,19 @@ With this default option each score is compiled as a sequence of PDF files
 representing one system each. By default the systems are separated by a
 paragraph and a variable skip depending on the staffsize.
 
+\lyIssue{Note:}
+\option{insert=systems} implies the use of `lilypond-book-preamble.ly`.
+It is worth pointing out that this will *not* have any notion of pages
+anymore - resulting in the staff-staff spacing to be minimal/natural.
+Usually LilyPond will space out the inter-staff (not -system!) space
+when the page is not filled, but with \option{insert=systems}
+this will not happen.
+While this behavior is usually desirable when including score examples
+in text it may result in suboptimal output for multi-page scores when there's
+the typical issues of how many systems will fit on a page.
+Also notice that by default pages will be ragged-bottom,
```

Comment:
Suggestion: insert something like “and LilyPond will not make any efforts to optimize page breaks.“